pkg/controller/manage/manage_controller.go

```diff
@@ -333,10 +295,39 @@ func (c *KmeshManageController) processItems() bool {
 		}
 		return true
 	}
+
 	c.queue.Forget(key)
 	return true
 }
 
+func (c *KmeshManageController) syncPod(key QueueItem) error {
+	pod, err := c.podLister.Pods(key.podNs).Get(key.podName)
+	if err != nil {
+		if apierrors.IsNotFound(err) {
+			return nil
+		}
+		log.Errorf("failed to get pod %s/%s: %v", key.podNs, key.podName, err)
```

Comment:
This error message is presented on line L290. My suggestion is to delete the log here and put the message in the err return value.
```suggestion
		return fmt.Errorf("failed to get pod %s/%s: %v", key.podNs, key.podName, err)
```